src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,151 @@
+package Duke;
+
+import Duke.task.Deadline;
+import Duke.task.Event;
+import Duke.task.Task;
+import Duke.task.Todo;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class Duke {
+
+    private static void writeToFile(String filePath, Integer cnt, ArrayList list) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        for (int i = 0; i < cnt; i++) {
+            fw.write(list.get(i).toString() + System.lineSeparator());
+        }
+        fw.close();
+    }
+
+    private static void getFileContents(String filePath, ArrayList list) throws FileNotFoundException {
+        File f = new File(filePath);
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            list.add(s.nextLine());
+        }
+    }
+    public static void main(String[] args) throws EmptyToDo, IllegalCommand {
+
+        String line = "____________________________________________________________\n";
+        Scanner cmd = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<>();
+        int cnt = 0;
```

Comment:
Maybe this name is not very self-explanatory? 

https://nus-cs2113-ay2223s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-name-to-explain